### ISSUE TITLE
Question about the baseline accuracy on Cifar-10

### ISSUE THREAD

**Issue by Changes92** [XzwHan/CARD#4]
Thanks for your great work!

I have a small question about the results in Table 4. I run the code and find the ResNet-18 ACC on Cifar-10 is around 94% (as the results in [this repo.](https://github.com/kuangliu/pytorch-cifar) ), which is significantly better than all methods in Table 4. Is the Table 4 experiment conducted in different environments? What causes this performance gap? @XzwHan

**Comment by XzwHan** [comment]
Hi @tmp12316, thank you for your question!

The repo you pointed to demonstrates the accuracy of a deterministic classifier, corresponding to the $f_{\phi}$ network in our paper. The accuracy on CIFAR-10 depends a lot on the training scheme (learning rate schedule, training epochs, etc.); we found [here](https://paperswithcode.com/sota/image-classification-on-cifar-10?tag_filter=93) that it has reached $95.55\\%$ for the ResNet-18 architecture with their design for training.

As pointed out at the beginning of Section 4.2 in our paper, we do not aim to achieve the SOTA results in terms of ACC; instead, we hope to promote the framework of training and inference with a stochastic classifier. As we've observed through our experiments, no matter what accuracy we can achieve with $f_{\phi}$, CARD would always obtain a better accuracy with majority vote. The point is that stochasticity could be applied to improve the accuracy of a baseline deterministic classifier. For the CIFAR-10 dataset, we only trained $f_{\phi}$ for 10 epochs (Appendix A.8 for details) and obtained $90.39\\%$ ACC; we improved it to $90.93\\%$ with CARD. Similar observations can be made on all other datasets. We expect that once $f_{\phi}$ is replaced with a better one, CARD would achieve a better ACC as well, with the improvement upon $f_{\phi}$.

**Comment by XzwHan** [comment]
Hi @tmp12316, we've come across Issue #7, and realized what you meant by the accuracy being ~ $94\\%$ -- please check our reply in that issue, which should apply to your question as well. Apologize for not fully getting your question at the time; hope this helps.
